src/Controller/ViewerController.php

```diff
@@ -63,63 +63,42 @@ public function __construct(
    *   TRUE to open Collabora Online in edit mode.
    *   FALSE to open Collabora Online in readonly mode.
    *
+   * @throws \Symfony\Component\HttpKernel\Exception\BadRequestHttpException
+   *   The Collabora server or editor is not available, or mismatching scheme.
+   *
    * @return \Symfony\Component\HttpFoundation\Response
    *   Response suitable for iframe, without the usual page decorations.
```

Comment:
We typically put `@throws` below `@return`.
```suggestion
   * @return \Symfony\Component\HttpFoundation\Response
   *   Response suitable for iframe, without the usual page decorations.
   *
   * @throws \Symfony\Component\HttpKernel\Exception\BadRequestHttpException
   *   The Collabora server or editor is not available, or mismatching scheme.
```